config/ompi_setup_prrte.m4

```diff
@@ -195,6 +203,9 @@ AC_DEFUN([_OMPI_SETUP_PRRTE_INTERNAL], [
     opal_prrte_CPPFLAGS_save="${CPPFLAGS}"
     OPAL_FLAGS_APPEND_UNIQ([CPPFLAGS], [${opal_pmix_CPPFLAGS}])
 
+    AC_DEFINE_UNQUOTED([OMPI_HAVE_PRTE_LAUNCH], [1],
```

Comment:
This is a second, seemingly redundant, AC_DEFINE_UNQUOTED for `OMPI_HAVE_PRTE_LAUNCH`.

Also, is the comment correct -- is prte_launch only available in internal prte?  It looks like you're testing for `prte_launch` in the external case, too.

Reply:
reworking this.  i figured we maywant to support the case that the external prrte has ```prte_launch```